### PR TITLE
Add the possibility of setting the value range manually in the Signal Strength widget configuration

### PR DESCRIPTION
**Pull request by jktu2870**
## Before
How to demo somethings that hadn't existed before?

## After
Peek 2024-02-15 14-06

## Pull Request description

The Signal Strength widget has some limitations. 
Currently, a value of -100 is the final acceptable value to display on the widget. A value of -100 is hardcoded and is not displayed on the chart. 

The customer has a scenario where the value displayed on the widget may be outside of this range (e.g., -130).

Therefore, the customer would like to be able to manually set the values for each range in the way that there is a possibility to configure the color of active/inactive signal bars under the 'Signal strength card style' section:

https://thingsboard-portal.atlassian.net/browse/PROD-2773 

## General checklist

- [ ] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [ ] [Labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-labels) that classify your pull request have been added.
- [ ] The [milestone](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) is specified and corresponds to fix version.  
- [ ] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [ ] Description contains human-readable scope of changes.
- [ ] Description contains brief notes about what needs to be added to the documentation.
- [ ] No merge conflicts, commented blocks of code, code formatting issues.
- [x] Changes are backward compatible or upgrade script is provided.
- [ ] Similar PR is opened for PE version to simplify merge. Crosslinks between PRs added. Required for internal contributors only.
  
## Front-End feature checklist

- [x] Screenshots with affected component(s) are added. The best option is to provide 2 screens: before and after changes;
- [x] If you change the widget or other API, ensure it is backward-compatible or upgrade script is present.
- [ ] Ensure new API is documented [here](https://github.com/thingsboard/thingsboard-ui-help)

### CODE REVIEW

**Comment by vvlladd28** [comment]
No setting was given to component tb-signal-strength-widget-settings.

File: ui-ngx/src/app/modules/home/components/widget/config/basic/indicator/signal-strength-basic-config.component.html [hunk 1]
```diff
@@ -109,6 +109,13 @@
                       formControlName="inactiveBarsColor">
       </tb-color-input>
     </div>
+    <div class="tb-form-row space-between">
+      <div translate>widgets.signal-strength.no-signal-rssi-value</div>
+      <mat-form-field appearance="outline" class="number" subscriptSizing="dynamic">
+        <input matInput formControlName="noSignalRssiValue" type="number" step="1"
+               placeholder="{{ 'widget-config.set' | translate }}">
+      </mat-form-field>
+    </div>
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/components/widget/config/basic/indicator/signal-strength-basic-config.component.html:118]: Perhaps this setting does not need to be added to the basic form.

**Review by vvlladd28 — CHANGES_REQUESTED** [review]

File: ui-ngx/src/app/modules/home/components/widget/config/basic/indicator/signal-strength-basic-config.component.html (hunk 1 above)
Comment (jktu2870) [reply ui-ngx/src/app/modules/home/components/widget/config/basic/indicator/signal-strength-basic-config.component.html:118]: Advanced form is indeed a better place for this setting

**Comment by jktu2870** [comment]
I have ruined this PR, created a new one https://github.com/thingsboard/thingsboard/pull/10226

**Pull request closed without merging**